kafka-ui-react-app/src/widgets/ClusterConfigForm/index.tsx

```diff
@@ -75,7 +75,7 @@ const ClusterConfigForm: React.FC<ClusterConfigFormProps> = ({
   const onReset = () => methods.reset();
 
   const onValidate = async () => {
-    await trigger();
+    await trigger(undefined, { shouldFocus: true });
```

Comment:
with `undefined` as the first argument, `trigger` validates all fields provided in `useForm` config